src/SkyApm.Utilities.Configuration/ConfigurationFactory.cs

```diff
@@ -67,6 +67,8 @@ public IConfiguration Create()
 
             builder.AddEnvironmentVariables();
 
+            builder.AddConfiguration(_configuration);
```

Comment:
should perform null check before call the AddConfiguration() method, becase for the AspNet agent, the passed in _configuration is null.

Reply:
Oh, i feel so sorry, i have committed